.github/pull_request_template

```diff
@@ -0,0 +1,36 @@
+Thanks for your contribution to the LEDE project! 
+To help people review your contribution we ask you to follow these rules:
+
+- use a different git branch for each pull request (GitHub does this automatically if you edit files through the 
+  website)
+
+- commit subject must start with "package name:" for packages, "device architecture:" for devices and "tool name:" 
+  for tools 
+- commit subject must be no more than 70-75 characters, and it must describe what the commit changes and why
+  the commit might be necessary. It is challenging to be both succinct and descriptive, but that is what a
+  well-written summary should do.
+
+- commit description will be committed to the source changelog, so it should make sense to a competent reader 
+  who does not know why you made this commit.  Including symptoms of the failure which you are fixing with 
+  this commit (log messages, error messages, etc.) is useful for people who might be searching the commit logs 
+  looking for a fix for their issue.  
+  If a patch fixes a compile failure, include only the part of the error log that will allow someone 
+  to find this fix. As for the commit subject, it is important to be both succinct as well as descriptive.
```

Comment:
` As for the commit subject, it is important to be both succinct as well as descriptive.`
This sentence is a loose duplicate of the similar text on lines 10-11. Contextually this would fit better to lines 9-11 where commit subject is discussed, instead of here after you have been talking about the longer contents and error log extracts etc. to be included in the commit message. So, I propose to delete this sentence from here.